modules/meta.analysis/tests/testthat/test.run.meta.analysis.R

```diff
@@ -1,42 +1,11 @@
-#-------------------------------------------------------------------------------
-# Copyright (c) 2012 University of Illinois, NCSA.
-# All rights reserved. This program and the accompanying materials
-# are made available under the terms of the 
-# University of Illinois/NCSA Open Source License
-# which accompanies this distribution, and is available at
-# http://opensource.ncsa.illinois.edu/license.html
-#-------------------------------------------------------------------------------
+test_that("`runModule.run.meta.analysis`", {
```

Comment:
note that the argument to `test_that` should be a description of the tests within. The `context` function can name the function or set of functions being tested.

Reply:
The testthat docs now discourage use of `context` and recommend encoding that info in the filename